lib/bot/setAudience.js

```diff
@@ -47,7 +47,7 @@ function setUserGroup(bot, message) {
 }
 
 function attachListener(controller) {
-  controller.hears(['(usergroup|audience) (\\S*)'],['direct_mention'], setUserGroup);
+  controller.hears(['(usergroup|audience) (<!(subteam|here|channel).*>)'],['direct_mention'], setUserGroup);
```

Comment:
Should `.*` be set to be non-greedy, to avoid the unlikely scenario where there's a `>` somewhere later in the message?